R/ppc-discrete.R

```diff
@@ -298,34 +282,69 @@ ppc_rootogram <- function(y,
       color = get_color("lh"),
       linewidth = 0.25,
       width = 1
-    ) +
-    bayesplot_theme_get()
-
-  if (style != "standing") {
-    graph <- graph + hline_0(size = 0.4)
+    )
   }
 
-  graph <- graph +
+  geom_yrep <- if (style == "discrete") {
+    geom_pointrange(
+      aes(y = .data$pred_median, ymin = .data$lower, ymax = .data$upper, color = "y_rep"),
+      fill = get_color("lh"),
+      linewidth = size,
+      size = size,
+      fatten = 2,
+      alpha = 1
+    )
+  } else {
     geom_smooth(
-      aes(
-        x = .data$xpos,
-        y = .data$tyexp,
-        color = "Expected"
-      ),
+      aes(x = .data$xpos, y = .data$tyexp, color = "Expected"),
       fill = get_color("d"),
       linewidth = size,
       stat = "identity"
-    ) +
-    scale_fill_manual("", values = get_color("l")) +
-    scale_color_manual("", values = get_color("dh")) +
-    labs(x = expression(italic(y)),
-         y = expression(sqrt(Count)))
-
-  if (style == "standing") {
-    graph <- graph + dont_expand_y_axis()
+    )
+  }
+
+  # Creating the graph
+  graph <- ggplot(data)
+
+  if (style == "discrete") {
+    graph <- graph +
+      geom_yrep +
+      geom_y +
+      aes(x = xpos) +
```

Comment:
This line is causing a NOTE in R cmd check (which doesn't cause it to fail on GitHub Actions but will be noticed by CRAN): 

https://github.com/stan-dev/bayesplot/actions/runs/17111544388/job/48533894818?pr=362#step:6:111

I think it probably needs to be `.data$xpos`